db/seeds/1_coffee_clifton.rb

```diff
@@ -1,5 +1,7 @@
+roaster = Roaster.find_by reference: 'clifton_coffee'
+
 Coffee.create!(
-  roaster_id: 'clifton-coffee-bristol',
+  roaster: roaster,
   name: "Banko Gotiti (Washed)",
   country: "Ethiopia",
   region: "Gotiti Kebele, Kochere District",
```

Comment:
`roaster.coffees.create!()` would be nicer here

Reply:
good call. done.